README.md

```diff
@@ -0,0 +1,65 @@
+# BiometricLocker
+
+### Example:
+Here's a simple example implementation through the AppDelegate.
+
+```swift
+import BiometricLocker
+
+@UIApplicationMain class AppDelegate: UIResponder, UIApplicationDelegate {
+  var biometricLocker = BiometricLocker()
+  
+  func application(_ application: UIApplication, didFinishLaunchingWithOptions options: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+    self.lockAppIfNeeded()
+    
+    ///…
+    
+    return true
+  }
+
+  func applicationDidEnterBackground(_ application: UIApplication) {
+    self.biometricLocker.lock()
+  }
+	
+  func applicationWillEnterForeground(_ application: UIApplication) {
+    self.lockAppIfNeeded()
+  }
+	
+  func applicationWillTerminate(_ application: UIApplication) {
+    var backgroundTask = UIBackgroundTaskInvalid
+
+    backgroundTask = application.beginBackgroundTask {
+      application.endBackgroundTask(backgroundTask)
+      backgroundTask = UIBackgroundTaskInvalid
+    }
+
+    DispatchQueue.global(qos: .default).async {
+      // Ensures the app is locked, regardless of our timeout allowance.
+      self.biometricLocker.deactivate(Date.distantPast)
```

Comment:
I don't believe there's a `deactivate` method here still - could you please update this documentation?